README.md

```diff
@@ -7,87 +7,108 @@ This repository's `master` branch tracks images pushed to the official Demisto D
+
+When opening a new pull request, a bot will evaluate whether you have signed the CLA. If required, the bot will comment on the Pull Request, including a link to accept the agreement. The CLA document is also available for review as a [PDF](https://github.com/demisto/content/blob/master/docs/cla.pdf).
 Visit our [Frequently Asked Questions](https://xsoar.pan.dev/docs/concepts/faq#cla-is-pending-even-though-i-signed-the-agreement) article for CLA related issues.
 
-After opening your docker pull request, and in order for the reviewer to understand the context, make sure to link to the corresponding pull request from the [Content](https://github.com/demisto/content) repo where this docker image will be used.
-## Getting Started
-Each docker image is managed in its own directory. The directory should be named the same as the image name (without the organization prefix). If needed, we prefer using a dash (`-`) as a separator in the name. All image directories are located under the `docker` dir.
+After opening a Pull Request, and in order for the reviewer to understand the context, make sure to link to the corresponding Pull Request from the [Content](https://github.com/demisto/content) repo where this Docker image will be used.
 
-The directory should contain one Dockerfile which will be used for building the docker image. Each image when it is built is tagged with the commit hash and version. 
 
-The script `docker/build_docker.sh` is used to build all modified docker images. The script detects modified directories by comparing against origin/master if on a branch or if on master by using the CIRCLE_COMPARE_URL environment variable to obtain the commit range of the current build.
+### How to Contribute
 
-**Pre-requisites:**
-* Install python 2 and 3 (so you can create both python 2 and 3 images):
-  * Mac: use brew to install (more info at: https://docs.brew.sh/Homebrew-and-Python): `brew install python3` and then: `brew install python@2`
-* Or install pyenv (recommended for managing multiple python versions):
-    * Mac: `brew install pyenv` . Make sure to run then: `pyenv init` and follow instructions to add to either `~/.zshrc` or `~/.bash_profile` depending on you shell.
-    * Other: see https://github.com/pyenv/pyenv-installer 
-* Install pipenv globally using: `pip install pipenv`
-* Install requests globally: `pip install requests`
+#### Prequisites
+Make sure you meet the following prerequisites:
+* An active GitHub account.
+* A [clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) or a [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) of the repository. The main difference is that if the contribution is done from a fork, the development Docker image created during the Pull Request build will not be deployed to DockerHub whereas opening a Pull Request from a clone will deploy the image.
```

Comment:
I think this isnt correct. The only way to contribute is from a fork. Cloning the repository just means to make a local copy, but the commits are done in the same repo.
Contributors arent allowed to write to our repo at all, so currently there isnt a way to deploy their changes to our dockerhub account (we have a task open for this, lets discuss)

The current solution is to follow the build output which provides a command to run in order to load the image locally. I would add this note here